### ISSUE TITLE
EmbededPythonHome with spaces does not allow pip to install

### ISSUE THREAD

**Issue by iongion** [henon/Python.Included#59]
I currently have this

```
"'C:\\ProgramData\\My' is not recognized as an internal or external command,\r\noperable program or batch file.\r\n"
```

But the real path is

```
"'C:\\ProgramData\\My Awesom Tool' is not recognized as an internal or external command,\r\noperable program or batch file.\r\n"
```

These need escaping

- ` RunCommand(text + " install " + module_name + version + " " + text2);`
- `RunCommand("cd " + EmbeddedPythonHome + " && python.exe Lib\\get-pip.py");`

Maybe some useful links

- <https://learn.microsoft.com/en-us/archive/blogs/twistylittlepassagesallalike/everyone-quotes-command-line-arguments-the-wrong-way>
- <https://stackoverflow.com/questions/5510343/escape-command-line-arguments-in-c-sharp>

**Comment by henon** [comment]
Check out the commit, is that what you mean? [83dc0a2](https://github.com/henon/Python.Included/commit/83dc0a2d6753ca11410690e97e4cd71b49fcaaa9)

**Comment by iongion** [comment]
Awesome, I guess that should be it, testing tomorrow, huge thank you!

**Comment by henon** [comment]
I packed new nugets:
https://www.nuget.org/packages/Python.Included/3.11.5
https://www.nuget.org/packages/Python.Deployment/2.0.4

If there are any issues you can reopen this thread.

**Comment by henon** [comment]
For your info, the adding of quotes caused a follow-up error which would cause pip not to install due to missing quotes around the cmd.exe /C argument. this is fixed now too.

**Comment by henon** [comment]
https://www.nuget.org/packages/Python.Deployment/2.0.5
https://www.nuget.org/packages/Python.Included/3.11.6
